doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSets/editing-data-sets-proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='editing-data-sets-proc_{context}']
+= Editing data sets
+
+You can edit existing data sets to ensure that the data fetched to the reporting components is up-to-date.
+
+.Procedure
+. In {CENTRAL}, go to *Admin* -> *Data Sets*.
++
+The *Data Set Explorer* page opens.
+. In the *Data Set Explorer* pane, search for the data set you want to edit and click *Edit*.
```

Comment:
In order to see the "Edit" button, you need to clikc on the data set first. So search for the data set -> click on the data set -> click Edit button

Reply:
Done